service/controller/v13/resource/cloudformation/resource.go

```diff
@@ -95,6 +102,58 @@ func (r *Resource) getCloudFormationTags(customObject v1alpha1.AWSConfig) []*aws
 	return stackTags
 }
 
+func (r *Resource) getStackOutputs(ctx context.Context, stackName string) ([]*awscloudformation.Output, error) {
+	input := &awscloudformation.DescribeStacksInput{
+		StackName: aws.String(stackName),
+	}
```

Comment:
I think we could use the cloudformation service to get the output.

https://github.com/giantswarm/aws-operator/blob/bbbbdd2fce90b8cfd843e0321cf1fb91fd0af897/service/controller/v13/cloudformation/cloud_formation.go#L76

Reply:
+1 we already do this in `GetCurrentState` we should leverage the information we already fetched instead of doing another call.

Reply:
Absolutely, didn't know about the CF service. Thanks for pointing it out, things are simpler using it, PTAL.